main.go

```diff
@@ -51,7 +51,10 @@ var noWebService = flag.Bool("noweb", false, "No web service")
 var skipVerify = flag.Bool("skipVerify", false, "Skip certificate verification")
 
 // 自定义 DNS 服务器
-var customDNS = flag.String("dns", "", "Custom DNS server, example: 8.8.8.8")
+var customDNS = flag.String("dns", "", "Custom DNS server address, example: 8.8.8.8")
+
+// 重置密码
+var newPassword = flag.String("password", "", "Reset password")
```

Comment:
```suggestion
var resetPassword = flag.Bool("reset-password", false, "Reset password")
```

如果不看描述的话，看不出来这个标志到底是用来重置密码，还是用传递的密码登录的。

可以改为使用 bool 标志，然后随机生成一个密码或者从 stdin 读取。

Reply:
README.md中有说明，也有描述。
`改为使用 bool 标志，然后随机生成一个密码或者从 stdin 读取。` 感觉稍微复杂了

Reply:
描述：`Reset password, example: ./ddns-go -password 123456`?

Reply:
> 描述：`Reset password, example: ./ddns-go -password 123456`?

重置密码是个低频操作，把标志名改成 `resetPassword` 应该就够了。

```
-resetPassword string
        Reset password to the one entered
```

Reply:
可以